src/components/OptionList/components/Option/Option.tsx

```diff
@@ -111,13 +111,7 @@ export function Option({
   );
 
   return (
-    <li
-      key={id}
-      className={optionClassName}
-      tabIndex={-1}
-      aria-selected={active}
-      role={role}
-    >
+    <li key={id} className={optionClassName} tabIndex={-1} role={role}>
```

Comment:
I had to remove `aria-selected={active}` as it was not valid on an `li`. I am worried this will have an adverse affect when the child is a `Button`.

Reply:
What did the screenreader read out?